config/application.rb

```diff
@@ -238,6 +238,8 @@ def config.database_configuration
     # Our templates shouldn't start with 'discourse/templates'
     config.handlebars.templates_root = 'discourse/templates'
     config.handlebars.raw_template_namespace = "Discourse.RAW_TEMPLATES"
+    Sprockets.register_mime_type 'text/x-handlebars', extensions: ['.hbr'] # or other extension which you like.
```

Comment:
Did you mean that leave that comment?

```suggestion
    Sprockets.register_mime_type 'text/x-handlebars', extensions: ['.hbr']
```